### PR TITLE
Comment controller repo model

### PR DESCRIPTION
**Pull request by ackley787**
updated pull request

### CODE REVIEW

**Review by scald97 — COMMENTED** [review]
Looks good to me, second to merge if agree

**Comment by acolter12** [comment]
Hi, this is giving me errors, I'm not sure if this is just a problem for me on my side. Can someone else can check out this code on their computer and see if they get any errors?

File: application/src/main/java/org/wecancodeit/serverside/model/Comment.java [hunk 1]
```diff
@@ -1,4 +1,45 @@
...
+    public String getUser() {
+        return user;
+    }
+
+    public Comment(){
+
+    }
+
+    public Comment(Long id, String comment, String post, String topic, String userName) {
+        this.id = id;
```
Comment (JamieCBarr) [inline application/src/main/java/org/wecancodeit/serverside/model/Comment.java]: Remember, we don't need to include id here, it's a generated value Spring will handle for us

File: application/src/main/java/org/wecancodeit/serverside/controller/CommentController.java [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
+@RestController
+@CrossOrigin
 public class CommentController {
+
+    @Resource
+    private CommentRepo commentRepo;
+
+    @GetMaping("/api/comment")
+    public Collection<CommentCollection> getUserNames(){
+        return (Collection<CommentCollection>) userNameRepo.findAll();
```
Comment (JamieCBarr) [inline application/src/main/java/org/wecancodeit/serverside/controller/CommentController.java]: This just needs to be Collection<Comment>, and this also seems like it wants to be calling commentRepo.findAll()

File: application/src/main/java/org/wecancodeit/serverside/controller/CommentController.java [hunk 3]
```diff
@@ -1,4 +1,53 @@
...
+    }
+
+    @PostMapping("/api/users/add-user")
+    public Collection<UserName> addUsers(@RequestBody String body) throws JSONException {
```
Comment (JamieCBarr) [inline application/src/main/java/org/wecancodeit/serverside/controller/CommentController.java]: this entire postMapping doesn't need to exist, don't worry about adding users here, that will be handled in the UserController

**Comment by acolter12** [comment]
Hi this is giving me errors still, I'm not sure if it has something to do with the imports or if its just my computer. I have included a screen shot of what I see on my end.

Screen Shot 2022-08-01 at 8 19 33 PM

**Pull request closed without merging**